### ISSUE TITLE
RecyclerView does not accept custom animations

### ISSUE THREAD

**Issue by djnotes** [leochuan/ViewPagerLayoutManager#39]
Hey 
I was trying to apply custom animations to the items of the list using ConstraintLayout animations (an example [here](https://github.com/djnotes/ConstraintLayoutAnimation)). It had no effect. I also tried another library ([Bungee](https://github.com/Binary-Finery/Bungee)). Again didn't work. It seems that VPLM disables all animations. What's your comment on this? Is it impossible to add custom animations to my activity and views when using VPLM?

**Comment by leochuan** [comment]
I don't think this kind of animation will effect the item of `recyclerView`, Does `linearLayoutManager` has the effect？If it does, maybe the animation will be supported in the future.

**Comment by djnotes** [comment]
Sorry Bungee had nothing to do with VPLM. It only works following a `startActivity(Intent)` call. With ConstraintLayout animation (like first getting constraint set from a new layout, applying it to viewholder, and then starting animation with `TransitionManager.beginDelayedTransition(constraintlayout)`, however, I am not sure.
